manifest.toml

```diff
@@ -0,0 +1,213 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "gitea"
+name = "Gitea"
+description.en = "Lightweight Git forge"
+description.fr = "Forge Git légère"
+
+version = "1.20.5~ynh2"
+
+maintainers = ["Josué Tille"]
+# previous_maintainers = [
+#     "rafi59"
+# ]
+
+[upstream]
+license = "MIT"
+website = "https://gitea.io/"
+admindoc = "https://docs.gitea.io/"
+code = "https://github.com/go-gitea/gitea"
+cpe = "cpe:2.3:a:gitea:gitea"
+fund = "https://opencollective.com/gitea"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = [
+    "amd64",
+    "i386",
+    "armhf",
+    "arm64"
+]
+multi_instance = true
+ldap = true
+sso = false
+disk = "120M"
+ram.build = "100M"
+ram.runtime = "100M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/gitea"
+
+    [install.admin]
+    type = "user"
+
+    [install.init_main_permission]
+    help.en = "Gitea should be visible by visitors to use git via HTTP and to use the API. You can still set repositories as private individually."
+    help.fr = "Gitea doit être visible par les visiteurs pour utiliser git via HTTP et utiliser l'API. Vous pouvez toujours rendre les dépôts privés individuellement."
+    type = "group"
+    default = "visitors"
+
+[resources]
+
+    [resources.system_user]
+    # Hard-coded...
+    home = "/home/yunohost.app/__APP__"
```

Comment:
what about adding `allow_ssh = true` as well as `allow_email = true` to system_user? (app.ini should be adapted with mail user and password)

Reply:
Oh ! That means I won’t have to manually add the user to app.ssh ?

Reply:
https://yunohost.org/fr/packaging_apps_resources#system-user

Reply:
Fixed.